app/uk/gov/hmrc/gform/upscan/UpscanService.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.upscan
+
+import akka.util.ByteString
+import java.time.Instant
+import scala.concurrent.{ ExecutionContext, Future }
+
+class UpscanService(
+  upscanConnector: UpscanConnector,
+  upscanRepository: UpscanRepository
+)(implicit ec: ExecutionContext)
+    extends UpscanAlgebra[Future] {
+
+  def download(
+    downloadUrl: String
+  ): Future[ByteString] = upscanConnector.download(downloadUrl)
```

Comment:
What is the maximum size of file? This can blow up whole service.

Reply:
Good observation. Any limit at this moment is imposed by upscan only.

There are two reasons why this is done.

 1. This implementation is prototype only, not everything is mean to stay for long.
 2. Following ticket is about taking this image and compressing it. And as far as I know there is no streaming compression solution for JVM, all implementations I've found are `Array[Byte] => Array[Byte]`. 
 
 To elaborate on point 1., ultimate goal is to spin up a new service which will be doing transferring/compressing of images and uploading them to `file-upload`